tests/test_backends.py

```diff
@@ -587,3 +595,57 @@ def testAcrossVariantSets(self):
                         self.verifySearchVariants(
                             variantSets, referenceName, 0, 2 ** 32,
                             pageSize=pageSize)
+
+
+class TestCallSets(TestWormtableBackend):
+    """
+    Tests the searchCallSets end point
+    """
+    def testCallSetIds(self):
+        """
```

Comment:
Unit test functions shouldn't have docstring comments --- I'm updating the style document now with some examples and the rationale for this.